### PR TITLE
Provide hover for ConfigProperty name

### PR DESCRIPTION
**Pull request by xorye**
The branch for this PR has been branched off of https://github.com/angelozerr/vscode-quarkus/tree/codelens, so it would be nice if https://github.com/redhat-developer/vscode-quarkus/pull/146 is merged before this PR is merged.

This PR works alongside:

### CODE REVIEW

File: src/definitions/constants.ts [hunk 1]
```diff
@@ -33,6 +35,7 @@ export namespace VSCodeCommands {
   export const ADD_EXTENSIONS = 'quarkusTools.addExtension';
   export const DEBUG_QUARKUS_PROJECT = 'quarkusTools.debugQuarkusProject';
   export const QUARKUS_WELCOME = 'quarkusTools.welcome';
+  export const QUARKUS_OPEN_URL = 'quarkus.open.url';
```
Comment (angelozerr) [inline src/definitions/constants.ts]: Why you need that?

File: src/extension.ts [hunk 2]
```diff
@@ -130,6 +140,13 @@ function registerVSCodeCommands(context: ExtensionContext) {
     WelcomeWebview.createOrShow(context);
   }));
 
+  /**
+   * Command for opening CodeLens URL
+   */
+  context.subscriptions.push(commands.registerCommand(VSCodeCommands.QUARKUS_OPEN_URL, (url) => {
```
Comment (angelozerr) [inline src/extension.ts]: Why you need that?

Comment (xorye) [reply src/extension.ts]: Looks like it is some old code from before, I will remove it

File: src/definitions/constants.ts (hunk 1 above)
Comment (xorye) [reply src/definitions/constants.ts]: Looks like it is some old code from before, I will remove it

**Comment by xorye** [comment]
@angelozerr fixed

**Comment by angelozerr** [comment]
Thanks @xorye !

**Pull request merged**